ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -211,6 +251,8 @@ class TabBarItemView: UIControl {
 
     let badgeView: UILabel = BadgeLabel(frame: .zero)
 
+    let unreadDotView: UILabel = BadgeLabel(frame: .zero)
```

Comment:
why does it  have to be a label? if it does, then the badgeView can be used for unreaddot scenario as well?